aplib.net-demo/Assets/Scripts/Camera/CameraManager.cs

```diff
@@ -1,10 +1,12 @@
+using System;
 using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
 
 public class CameraManager : MonoBehaviour
 {
     [SerializeField] public Transform PlayerVisTransform;
+    public Transform PlayerTransform;
```

Comment:
It is better to use everything as it was, and just to rename the PlayerVisTransform to PlayerTransform, to avoid issues and complexity as it does not get used anywhere else.